apache2/modsecurity.c

```diff
@@ -153,85 +169,53 @@ int modsecurity_init(msc_engine *msce, apr_pool_t *mp) {
 #ifdef WITH_CURL
     curl_global_init(CURL_GLOBAL_ALL);
 #endif
+    lock_args = apr_pcalloc(mp, sizeof(struct waf_lock_args));
+
     /* Serial audit log mutext */
-    rc = apr_global_mutex_create(&msce->auditlog_lock, NULL, APR_LOCK_DEFAULT, mp);
-    if (rc != APR_SUCCESS) {
+    lock_name = apr_pstrcat(mp, "Global\\", "auditlog_lock", NULL);
```

Comment:
It may be useless if you just run on Linux.

Reply:
will make modsec specific